developer/guides/troubleshooting/tailwind-styles-being-overridden.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Tailwind styles are being overridden
+description: Resolve issues with Tailwind v4 or other @layer-based CSS frameworks being overridden by CSS resets
+---
+
+## Problem
+
+Starting from [version 4](https://tailwindcss.com/blog/tailwindcss-v4-alpha#designed-for-the-modern-web), Tailwind CSS uses `@layer` rules to define its styles.
+
+Makeswift by default includes a CSS reset from [normalize.css](https://github.com/necolas/normalize.css/).
+Because the CSS reset is unlayered, it'll override the Tailwind styles, even if the Tailwind styles are more specific.
+
+From [MDN @layer documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/@layer):
+
+![Image: CSS layer cascade](/images/layer-cascade.svg)
+
+
+## Solution
+
+Disable the built-in CSS reset by setting `enableCssReset={false}` on the `RootStyleRegistry`:
```

Comment:
What impact does this have? I would assume this would mess with expected styles if the reset isn't applied? Is that something we really want to encourage?

Reply:
Yes. This is something we want to encourage if they're using a CSS framework like Tailwind since they come with their own CSS reset: https://tailwindcss.com/docs/preflight

I'll add this to the docs.